### PR TITLE
Update `RDT` to v1.0.0

### LINKED ISSUE #897: Update the RDT version to 1.0
Update the current `RDT` version to `1.0` and the new `API` that this presents. Also update `CTGAN` and `SDMetrics` to the new release candidates.

### PR DESCRIPTION
**Pull request by pvk-developer**
Resolves #897

### CODE REVIEW

**Review by amontanez24 — CHANGES_REQUESTED** [review]
Mostly minor comments and questions

File: sdv/metadata/table.py [hunk 1]
```diff
@@ -404,24 +404,25 @@ def _get_transformers(self, dtypes):
         for name, dtype in dtypes.items():
             field_metadata = self._fields_metadata.get(name, {})
             transformer_template = field_metadata.get('transformer')
-            if self._dtype_transformers[np.dtype(dtype).kind] is None:
-                transformers[name] = None
-
-            elif transformer_template is None:
+            if transformer_template is None:
                 transformer_template = self._dtype_transformers[np.dtype(dtype).kind]
```
Comment (amontanez24) [inline sdv/metadata/table.py:408]: you can probably add this as the default to the get on line 406 so you only have to do this check once. ie
```python
transformer_template = field_metadata.get('transformer', self._dtype_transformers[np.dtype(dtype).kind])
```

**Review by amontanez24 — APPROVED** [review]
One more minor comment, but after that looks ready to go to me!

**Review by fealho — APPROVED** [review]
Looks good! Just double check why the tutorials are failing.

**Pull request merged**